edxval/utils.py

```diff
@@ -0,0 +1,27 @@
+"""
+Util methods to be used in api and models.
+"""
+
+from django.conf import settings
+from django.core.files.storage import get_storage_class
+
+
+def video_image_path(video_image_instance, filename):  # pylint:disable=unused-argument
+    """
+    Returns video image path.
```

Comment:
Let's add Arguments in this doc string and expectation about video_image_instance since it's not being used in this method which can confuse readers.